src/main/java/uk/gov/companieshouse/filinghistory/consumer/logging/LoggingKafkaListenerAspect.java

```diff
@@ -48,6 +49,9 @@ public Object manageStructuredLogging(ProceedingJoinPoint joinPoint) throws Thro
                     DataMapHolder.getLogMap());
 
             return result;
+        } catch (RetryableException ex) {
```

Comment:
We need to start including the number of attempts in the log map and log at error instead of info when we have completed the last attempt.

Reply:
Would a failure on the last attempt not get caught by the proceeding catch block?

Reply:
Added method to calculate number of retries.